src/api/app/components/chart_component.html.haml

```diff
@@ -5,7 +5,7 @@
       %i.fas.fa-sync-alt{ id: 'chart-build-reload' }
     %h4
       Build results summary
-    - if distinct_repositories.size > 5 || raw_data.size > 12
+    - if distinct_repositories.size > ChartComponent::MINIMUM_DISTINCT_BUILD_REPOSITORIES || raw_data.size > ChartComponent::MINIMUM_BUILD_RESULTS
```

Comment:
This change corresponds to changes coming from #, the should be added there and rebase this branch
https://github.com/openSUSE/open-build-service/pull/15221/files#diff-bd413799cfd90c78b4726ba5d29235fe39542e36ad8b79165986e8523c0664d0R5

Reply:
Well, yes and no. This line was changed in PR#15221, but the logic was already present even before. In the meanwhile that PR got merged :confused: .
I understand this change is out-of-scope of the current PR. I can extract this commit and open a separate PR if you prefer.

Reply:
LGTM

Reply:
No! Now that the other is merged, we can leave the change here. No worries